tools/noms/staging.py

```diff
@@ -28,36 +30,76 @@ def Main(projectName, stagingFunction):
         staging.Main('nerdosphere', staging.GlobCopier('index.html', 'styles.css', '*.js'))
     """
     parser = argparse.ArgumentParser(description='Stage build products from this directory.')
-    parser.add_argument('stagingDir',
+    parser.add_argument('staging_dir',
```

Comment:
Make sure every call site (script) has been updated. My comment was about the little closures that use `stagingDir` which doesn't affect the variable name. You might want to leave the flag the same.